### DISCUSSION TITLE
Home assistant crashes when installing integration

### DISCUSSION THREAD

**Discussion by Rebislori** [Tasshack/dreame-vacuum#114] [Q&A] [answered]
Hi everyone;

I'd like to thank all the people involved in this project, it's truly amazing.

This morning i migrated from dreamehome to mihome, installed the integration, everything is fine until after the login: the informations are retrieved correctly from the cloud, but when i click to "submit" ("invia", as i have home assistant in italian) home assistant reboots, or crashes; best case scenario: the integration remains stuck to "initializing" for half an hour.

Any idea about why this happens?

Thanks again.

**Comment by Tasshack** [comment, marked as answer]
Probably due to memory issue.
https://github.com/Tasshack/dreame-vacuum/issues/105
https://github.com/Tasshack/dreame-vacuum/issues/52

**Comment by Rebislori** [comment]
That was it; i installed it with no object in the map and everything works, thanks!

**Comment by Tasshack** [comment]
https://github.com/Tasshack/dreame-vacuum/issues/52#issuecomment-1692143931
